pages.es/osx/brew.md

```diff
@@ -0,0 +1,40 @@
+# brew
+
+> Administrador de paquetes para macOS.
+> Más información: <https://brew.sh>.
+
+- Buscar fórmulas y paquetes disponibles:
+
+`brew search {{text}}`
+
+- Instalar la última version estable de una fórmula (usar `--devel` para versiones de desarrollo):
+
+`brew install {{formula}}`
+
+- Mostrar todas las fórmulas instaladas:
+
+`brew list`
+
+- Mostrar fórmulas instaladas que no dependen de otras:
+
+`brew leaves`
+
+- Actualizar una fórmula instalada (si no se añade el nombre, se actualizarán todas las fórmulas):
```

Comment:
This text is not cool in English for me.
Creo que `indicar` es el verbo más semántico para esto:
```suggestion
- Actualiza una fórmula instalada (si no se indica el nombre, se actualizan todas las fórmulas):
```